### ISSUE TITLE
Document aggregation

### ISSUE THREAD

**Issue by antony** [mixmaxhq/mongo-cursor-pagination#54]
It looks like the aggregate method isn't documented in readme.md. Not sure if this was intentional or not.

**Comment by chaitu14341** [comment]
how to use the aggregate method ? May i know if you are already using aggregation with this module

**Comment by antony** [comment]
@chaitu14341 Yes I am using aggregate extensively. The docs are in the source code here https://github.com/mixmaxhq/mongo-cursor-pagination/blob/master/src/aggregate.js#L5

**Comment by sangdth** [comment]
> how to use the aggregate method ? May i know if you are already using aggregation using mongoose

No, with mongoose plugin there is only find method. I'm trying to add aggregate() into mongoose plugin but still not working as expected.

**Comment by nathanielrindlaub** [comment]
@sangdth, I am not seeing `aggregate.js` being required or used in the [mongoose.plugin.js](https://github.com/mixmaxhq/mongo-cursor-pagination/blob/master/src/mongoose.plugin.js). How were you able to use aggregate rather than `find() or `search()`?

**Comment by sangdth** [comment]
> 

Hi @nathanielrindlaub, unfortunately I forgot the context, it's 2020 my friend. But a quick look into the source code, I think you can't use aggregate() if you want to use it as Mongoose's plugin. Mongo and Mongoose are different things.

They export separated things in the index.js file, so I believe the last time I had to give up on using this code.
